pkg/kubelet/config/http.go

```diff
@@ -22,7 +22,7 @@ import (
 	"net/http"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
There is no need to use golang's package alias feature

Reply:
nit: Automatically added by formatting tool